frontend/src/app/modules/workspace/workspace-routing.module.ts

```diff
@@ -14,8 +15,8 @@ const routes: Routes = [
       {
         path: '',
         component: PersonalComponent,
-
       },
+      { path: 'inviteteam/:email', component: InviteFinishComponent },
```

Comment:
it won't work
I can manually go to inviteteam/{anythin} and it will perform some action and even show a message of success